### PR TITLE
Pract.03: Add 10-12 | Miroslav Stoyanovski

### PR DESCRIPTION
**Pull request by mir4oni3**
Miroslav Stoyanovski
6 5MI0600195

### CODE REVIEW

File: SI/Sem.03/Pract.03/Task11.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include <iostream>
```
Comment (dejansvetoslavov) [inline SI/Sem.03/Pract.03/Task11.cpp:1]: Is 0 Prime number?

File: SI/Sem.03/Pract.03/Task10.cpp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		}
+		if (n_len % 2 == 1) {
+			n_copy *= 10;
+		}
+		//n_copy is now n, but the last 4 digits are 0 instead of
+		//their usual values and the middle digits are removed
+		//(Ex. 1234567890 -> 12340000)
+		//(assuming n_len=10)
+		n_copy += n2;
+		cout << n_copy << ", " << (n_copy+1);
```
Comment (dejansvetoslavov) [inline SI/Sem.03/Pract.03/Task10.cpp:53]: Fix formatting. It is a bit hard to understand the code - try to refactor it in order to improve it.

File: SI/Sem.03/Pract.03/Task12.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+            cout << "true" << endl;
+            return 0;
+        }
+        if (n % 2 == 1) {
+            break;
+        }
+        n /= 2;
+    }
+    cout << "false" << endl;
+
```
Comment (dejansvetoslavov) [inline SI/Sem.03/Pract.03/Task12.cpp:19]: Remove blank line

**Review by dejansvetoslavov — CHANGES_REQUESTED** [review]

**Comment by GeorgiTerziev02** [comment]
The correct solutions have already been uploaded
https://github.com/Justsvetoslavov/Introduction_to_programming/tree/master/SI/Sem.03/Pract.03

**Pull request closed without merging**